packages/stats-py/aqsha_stats/analyses/uji_wilcoxon.py

```
@@ -36,19 +35,18 @@ def run(df: pd.DataFrame, args: dict) -> dict:
         ],
     )
     try:
-        w_stat, p = stats.wilcoxon(a, b)
+        # `method="approx"` = normal approximation (SPSS parity): the reported p-value then
+        # matches the Z below and the "Asymp. Sig." label. Without it scipy auto-picks the
+        # EXACT method for small n, whose p-value would contradict an "Asymp." label.
+        result = stats.wilcoxon(a, b, method="approx")
     except ValueError as exc:
         from ..contract import AnalysisError
 
         raise AnalysisError("insufficient_data", f"Uji Wilcoxon tidak bisa dihitung: {exc}")
```

Ruff (0.15.20)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)